notebooks/community/model_garden/model_garden_tfvision_image_segmentation.ipynb

```diff
@@ -0,0 +1,941 @@
+{
```

Comment:
<p>if the model is fully trained during HPT, then just export the best HPT model vs training again.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>